src/components/posts/PostsList.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <div>
+
+    <v-list
+        class="py-0"
+        outlined
+        rounded
+        width="100%"
+        color="transparent"
+    >
+      <template v-for="(post, index) in posts">
+        <v-list-item
+            :key="index"
+        >
+          <v-list-item-content>
+            <v-card
+                class="pa-5"
```

Comment:
on smaller screens: decrease paddings and margin. MAYBE try to decrease reactions buttons and position tehm vertically so that there is more space for post content on mobile devices
<img width="1510" alt="Screenshot 2021-09-04 at 20 23 12" src="https://user-images.githubusercontent.com/45401007/132104536-5e92576f-ae13-4ae9-aeb9-b0b10b1895a9.png">